shared/ui/components/form/Form.js

```diff
@@ -0,0 +1,63 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+import { Button } from '@aragon/ui'
+
+const Form = ({
+  children,
+  className,
+  onSubmit,
+  submitText,
+  disabled,
+  errors,
+}) => {
+  return (
+    <form
+      className={className}
+      onSubmit={(e) => {
+        e.preventDefault()
+        onSubmit()
+      }}
+    >
+      <VerticalSpace />
+      {children}
+      <Button
+        type="submit"
+        style={{ userSelect: 'none', marginTop: '24px' }}
+        mode="strong"
+        wide
+        onClick={onSubmit}
```

Comment:
Now that we have the `onSubmit` on the form, we do not need the `onClick` on this button